mapping-for-navigation/_posts/0000-01-08-implicit-maneuvers.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Mapping implicit turn-restriction
+---
+
+
+## What are implicit turns?
```

Comment:
Does "implicit turns" make sense? Shouldn't this be "implicit turn restrictions" because the turn restriction is implicit but the turn is not implicit.